set.mm

```diff
@@ -18869,6 +18869,26 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
       RCPABFJKSPCIBCDLPBCEMNO $.
   $}
 
+  ${
+    sbimd.1 $e |- F/ x ph $.
+    sbimd.2 $e |- ( ph -> ( ps -> ch ) ) $.
+    $( Deduction substituting both sides of an implication.  (Contributed by
+       Wolf Lammen, 24-Nov-2022.) $)
+    sbimd $p |- ( ph -> ( [ y / x ] ps -> [ y / x ] ch ) ) $=
+      ( weq wi wa wex wsb imim2d anim2d eximd anim12d df-sb 3imtr4g ) ADEHZBIZS
+      BJZDKZJSCIZSCJZDKZJBDELCDELATUCUBUEABCSGMAUAUDDFABCSGNOPBDEQCDEQR $.
+  $}
+
+  ${
+    sbbid.1 $e |- F/ x ph $.
+    sbbid.2 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Deduction substituting both sides of a biconditional.  (Contributed by
+       NM, 30-Jun-1993.)  Remove dependency on ~ ax-10 and ~ ax-13 .  (Revised
```

Comment:
What is the purpose of expanding the revision reason to this detail?  The proof became longer, but that is justified because its axiom footprint is smaller.  Is there a need to express more?

Reply:
I think this information can be useful, especially since we're trying to chase dependencies on ax-13. For instance, one may want to locate proofs where one succesfully removed it, to take inspiration. Plus, it's not very long to state it (if it's only one or two axioms, it's not much longer than "reduce axiom usage").